.clang-format

```diff
@@ -14,10 +14,10 @@ BraceWrapping:
   AfterNamespace: false
   BeforeElse: false #[M15]
 IndentCaseLabels: false #[M09]
-IndentWidth:     4 #[M08]
+IndentWidth:     2 #[M08]
```

Comment:
TizenRT coding rule is 4, not 2. Please revert this and below.

Reply:
TizenRT coding rule uses tab not space. So I think it does not violate the rule.